apps/prairielearn/elements/pl-drawing/pl-drawing.py

```diff
@@ -503,11 +503,24 @@ def test(element_html: str, data: pl.ElementTestData) -> None:
     name = pl.get_string_attrib(
         element, "answers-name", defaults.element_defaults["answers-name"]
     )
+    gradable = pl.get_boolean_attrib(
+        element, "gradable", defaults.element_defaults["gradable"]
+    )
+    if not gradable:
+        return
+
     # Get raw correct answer
     a_tru = data["correct_answers"][name]
+    grid_size = pl.get_integer_attrib(
+        element, "grid-size", defaults.element_defaults["grid-size"]
+    )
+    tol = pl.get_float_attrib(element, "tol", grid_size / 2)
+    angtol = pl.get_float_attrib(
+        element, "angle-tol", defaults.element_defaults["angle-tol"]
+    )
 
     result = data["test_type"]
-    if result == "correct":
+    if result == "correct" and len(a_tru) > 0:
```

Comment:
Do we need an `allow-blank`?

Reply:
I don't think so. If you want to make the case for adding it, that should be a separate issue or PR.